### PR TITLE
Readme.fr

### PR DESCRIPTION
**Pull request by B0pol**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (user facing)

#### Description of the changes in your PR

- record videos
- create clones
- take over the world
Translate README
#### Fixes the following issue(s)

- 

#### Relies on the following changes

- 

#### APK testing 

debug.zip

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

~~I still need to fix navigation links, but that will be easier with GitHub~~ done

### CODE REVIEW

**Comment by mhmdanas** [comment]
I'm making a PR to fix #5051, so I think you should wait until it is merged so that you can translate what I added too.

File: README.fr.md
Comment (pihug12) [inline README.fr.md:51]:
```suggestion
* Choisir le lecteur permettant de regarder des vidéos
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:54]:
```suggestion
* Ouvrir une vidéo dans Kodi
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:84]:
```suggestion
Lorsque des modifications sont effectuées au code de NewPipe (soit grâce à de nouvelles nouveautés, soit à cause de corrections de bugs), une mise à jour finira par être disponible. Le format de version est x.xx.x . Pour obtenir les nouvelles versions, vous pouvez :
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:87]:
```suggestion
 3. Télécharger un fichier APK depuis les [publications GitHub (releases)](https://github.com/TeamNewPipe/NewPipe/releases) et installez l'appli. Les mises à jour seront disponibles dès qu'elles sont publiées par l'équipe NewPipe.
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:88]:
```suggestion
 4. Mettre à jour via F-Droid. C'est la méthode la plus lente, puisque F-Droid doit compiler l'application, signer l'APK, puis publier la mise à jour à ses utilisateurs.
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:132]:
```suggestion
Par conséquent, notre appli ne collecte rien sans votre consentement. La politique de confidentialité de NewPipe explique en détail quelles données sont envoyées et conservées lorsque vous signalez une erreur, ou commentez sur notre blog. Vous pouvez trouver ce document [ici](https://newpipe.net/legal/privacy/).
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:139]:
```suggestion
telle que publiée par la Free Software Foundation, soit la version 3 de la Licence, soit (à votre gré) toute version ultérieure.
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:102]:
```suggestion
Si vous aimeriez contribuer, veuillez regarder nos [notes de contribution](.github/CONTRIBUTING.md).
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:99]:
```suggestion
Que ce soit pour des idées, des traductions, des changements de design, une amélioration du code ou un changement majeur du code, de l'aide est toujours appréciée.
```

File: README.fr.md
Comment (pihug12) [inline README.fr.md:95]:
```suggestion
3. Téléchargez NewPipe depuis la nouvelle source et installer l'application
```

**Review by pihug12 — CHANGES_REQUESTED** [review]
There are some typos.

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:87]:
```suggestion
 3. Télécharger un fichier APK depuis les [publications GitHub (releases)](https://github.com/TeamNewPipe/NewPipe/releases) et installer l'appli. Les mises à jour seront disponibles dès qu'elles sont publiées par l'équipe NewPipe.
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:23]:
```suggestion
<b>METTRE NEWPIPE OU N'IMPORTE QUEL FORK SUR LE GOOGLE PLAY STORE VIOLE LEURS CONDITIONS GÉNÉRALES D'UTILISATION.</b>
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:90]:
```suggestion
Nous recommandons la méthode 2 à la plupart de nos utilisateurs. Les fichiers APK installés via la méthode 2 ou 3 sont compatibles, mais pas avec la méthode 4. Nous utilisons notre signature pour signer les fichiers APK, mais F-Droid utilise sa propre signature. La création d'un APK via la méthode 1 exclut entièrement une signature. Android compare les signatures lors des mises à jour pour des raisons de sécurité, c'est pourquoi ces méthodes sont incompatibles.
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:21]:
```suggestion
<b>AVERTISSEMENT : C'EST UNE VERSION BÊTA, PAR CONSÉQUENT VOUS POURRIEZ RENCONTRER DES BOGUES. SI C'EST LE CAS, VEUILLEZ OUVRIR UN TICKET VIA NOTRE DÉPÔT GITHUB.</b>
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:84]:
```suggestion
Lorsque des modifications sont effectuées au code de NewPipe (soit grâce à de nouvelles nouveautés, soit à cause de corrections de bogues), une mise à jour finira par être disponible. Le format de version est x.xx.x . Pour obtenir les nouvelles versions, vous pouvez :
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:60]:
```suggestion
* Afficher des informations générales à propos des chaînes
* Rechercher des chaînes
* Regarder les vidéos d'une chaîne
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:64]:
```suggestion
* S'abonner à des chaînes
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:92]:
```suggestion
Cependant, si vous voulez changer de source pour certaines raisons (par ex. une fonctionnalité principale de NewPipe ne fonctionne plus et F-Droid n'a pas encore publié la mise à jour), nous vous recommandons cette procédure :
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:93]:
```suggestion
1. Sauvegardez vos données via les Paramètres > Contenu -> Exporter la base de données, afin de conserver les abonnements, les listes de lecture et l'historique
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:95]:
```suggestion
3. Téléchargez NewPipe depuis la nouvelle source et installez l'application
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:109]:
```suggestion
Si vous aimez NewPipe, nous serions heureux de recevoir un don. Vous pouvez soit donner avec des bitcoins, soit via Bountysource ou soit via Liberapay. Pour plus d'informations, veuillez visiter notre [site web](https://newpipe.net/donate).
```

File: README.fr.md
Comment (TiA4f8R) [inline README.fr.md:131]:
```suggestion
Le projet NewPipe cherche à fournir une expérience anonyme et privée pour les services multimédias en ligne.
```

File: README.fr.md
Comment (XiangRongLin) [inline README.fr.md:9]: This need to be replaced with the github badge.
see #5278

**Review by XiangRongLin — CHANGES_REQUESTED** [review]

File: README.fr.md
Comment (TobiGr) [inline README.fr.md:19]: the newly added translations need to linked

**Review by TobiGr — CHANGES_REQUESTED** [review]

File: README.fr.md
Comment (B0pol) [reply README.fr.md:60]: On n’est pas obligé https://fr.wiktionary.org/wiki/chaine

**Pull request closed without merging**
